src/Documents.jl

```diff
@@ -134,6 +134,31 @@ immutable EvalNode
     result :: Any
 end
 
+# Navigation
+# ----------------------
+
+"""
+Element in the navigation tree of a document, containing navigation references
+to other page, reference to the [`Page`](@ref) object etc.
+"""
+type NavNode
+    page           :: Nullable{Compat.UTF8String} # unless null, should point to a valid Page in the document
```

Comment:
What it unclearly tries to say is that there should a `Page` object in `doc.internal.pages` that has this name. Needs rephrasing indeed.